normative/dtpr-stickers
# chore: Uses remote icons for taxonomy stickers

### Review App

- [Review App Sensor](https://dtpr-sticker-chore-icon-o84pnw.herokuapp.com/sensors/20)
- [Review App Sensor Print](https://dtpr-sticker-chore-icon-o84pnw.herokuapp.com/sensors/20/print)

### Task Description

- Remove local taxonomy icons and use github helpful places icons configured via env variable

### PR Description (Proposed changes, approach, gotchas, etc)

- [x] Adds env variable (`REACT_APP_ICONS_BASE_URL`) to set base icons url path
- [x] Adds env variable (`REACT_APP_ICONS_PLACEHOLDER_BASE_URL`) to set base placeholder icons url path, if no env provided it will use local existing images. If decided to hold placeholder images into remote path, please, create a `placeholders` folder and place each taxonomy placeholder into this folder with their respective name, which would be `access`, `purpose`, `tech`, etc
![image](https://user-images.githubusercontent.com/22888260/113521976-87f9ba00-9573-11eb-9870-f3b292ce2d33.png)
    So, the link for placeholder icon svg would be something like `https://raw.githubusercontent.com/Helpful-Places/dtpr/master/dtpr_icons/placeholders/tech.svg`
- [x] Removes all icons
- [x] Removes all Svg icon React components 
- [x] Uses remote icon svgs to display sensor taxonomy
- [x] Uses remote icon svgs to display sensor print page
     *OBS*: The github link for `purpose/ic_white` is not matching helpful places `master` branch icon version, which might cause some error on layout because it has `react fill black`, not `transparent`

### Steps to Manually Test

- Check the sensor page and make sure all icons are there, if not there should show placeholder
- Check print page
